relay-general/src/pii/processor.rs

```diff
@@ -162,6 +162,19 @@ impl<'a> Processor for PiiProcessor<'a> {
 
         Ok(())
     }
+
+    // Replay PII processor entry point.
+    fn process_replay(
+        &mut self,
+        replay: &mut Replay,
+        _meta: &mut Meta,
+        state: &ProcessingState<'_>,
+    ) -> ProcessingResult {
+        if let Some(user) = replay.user.value_mut() {
+            user.process_child_values(self, state)?;
+        }
+        Ok(())
```

Comment:
I may have given misleading advice here earlier. We want all children of replay to be processed, not just the user field:
```suggestion
        replay.process_child_values(self, state)
```

This should fix the assertion on the credit card number.

Reply:
This fixes the assertion on the user PII but not on the tags PII.  Progress!